cli/types/tests/kitchen-sink.ts

```diff
@@ -0,0 +1,1470 @@
+// Samples taken from the cypress kitchen sink example (https://github.com/cypress-io/cypress-example-kitchensink)
```

Comment:
is this file automatically passing dtslint because there are no `$ExpectType` assertions?

Reply:
`dtslint` will look for compilation errors as well - unless you use `// $ExpectError` on the previous line.

You can see the TS compilation error it caught here: https://circleci.com/gh/cypress-io/cypress/7985

Reply:
It is really cool - you can make assertions if you want errors for something.